chevah/empirical
# [#2482] Add prefix/suffix for createFolder and writeFileContent.

# Problem

It would be nice to add a prefix on temp folders so that when debuging is much easier to identify their purpose.
# Changes

Add prefix and suffix to tempFolder just like we have for tempFile

As a drive by add a helper method to write content to a file.
# How to test

reviewers: @alibotean 

check that changes make sense.

I had to release 0.30.1 since for first release I forgot to merge master.

thanks!
